README.md

```diff
@@ -0,0 +1,17 @@
+# SDK Extension for Rust stable
+
+This extension contains various components of the [Rust](https://www.rust-lang.org) stable toolchain.
+
+
+## Mold
+
+In order to use the (fast) [mold](https://github.com/rui314/mold) linker:
+
+1. Add "org.freedesktop.Sdk.Extension.llvm12" next to this extension in order to get clang
```

Comment:
I would mention that is only needed until gcc12 lands on the freedesktop runtime. Let me ask around when that can happen

Reply:
Seems there is no tarballs for gcc12 yet, so I would mention the llvm12 is needed until there is a release of gcc12. then we will have it it in fdo 22.08 once it is out

Reply:
Done

Reply:
I would recommend using llvm13 here which is latest version on flathub